[2024-12-04T10:08:35.573+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-04T10:08:35.596+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_read.consume_from_topic manual__2024-12-04T10:08:34.226376+00:00 [queued]>
[2024-12-04T10:08:35.607+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_read.consume_from_topic manual__2024-12-04T10:08:34.226376+00:00 [queued]>
[2024-12-04T10:08:35.610+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-04T10:08:35.626+0000] {taskinstance.py:2889} INFO - Executing <Task(ConsumeFromTopicOperator): consume_from_topic> on 2024-12-04 10:08:34.226376+00:00
[2024-12-04T10:08:35.632+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1240) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-04T10:08:35.634+0000] {standard_task_runner.py:72} INFO - Started process 1241 to run task
[2024-12-04T10:08:35.634+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kafka_read', 'consume_from_topic', 'manual__2024-12-04T10:08:34.226376+00:00', '--job-id', '213', '--raw', '--subdir', 'DAGS_FOLDER/dag2_Kafkas3.py', '--cfg-path', '/tmp/tmpuluq2n9l']
[2024-12-04T10:08:35.637+0000] {standard_task_runner.py:105} INFO - Job 213: Subtask consume_from_topic
[2024-12-04T10:08:35.687+0000] {task_command.py:467} INFO - Running <TaskInstance: kafka_read.consume_from_topic manual__2024-12-04T10:08:34.226376+00:00 [running]> on host 31ac0cc6adcb
[2024-12-04T10:08:35.782+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_read' AIRFLOW_CTX_TASK_ID='consume_from_topic' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T10:08:34.226376+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-04T10:08:34.226376+00:00'
[2024-12-04T10:08:35.784+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-04T10:08:35.809+0000] {base.py:84} INFO - Retrieving connection 'consume_kafka'
[2024-12-04T10:09:35.814+0000] {consume.py:167} INFO - Reached end of log. Exiting.
[2024-12-04T10:09:35.816+0000] {consume.py:182} INFO - committing offset at end_of_operator
[2024-12-04T10:09:35.848+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-04T10:09:35.849+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kafka_read, task_id=consume_from_topic, run_id=manual__2024-12-04T10:08:34.226376+00:00, execution_date=20241204T100834, start_date=20241204T100835, end_date=20241204T100935
[2024-12-04T10:09:35.891+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-04T10:09:35.911+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-04T10:09:35.915+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
